src/terraform/index_test.go

```diff
@@ -69,7 +69,7 @@ provider "aws" {
 }
 
 module "aws" {
-  source = "./aws"
+  source = "git@github.com:Originate/exosphere.git//src//terraform//modules//aws?ref=8786f912"
```

Comment:
> change individually of each other

I don't know if I understand this. Why would we keep some referencing older versions of the codebase? If we only changed one, whats the harm with updating all of them?